shared/AppInsightsCore/Tests/Selenium/CorePerfCheck.Tests.ts

```diff
@@ -719,7 +717,7 @@ export class CorePerfCheckTests extends AITestClass {
             let devAllowance = (theDuration.deviation / 2);
             if ((theDuration.duration + devAllowance) <= maxTime && (theDuration.iteration < maxIteration || (theDuration.avgIteration - devAllowance) < maxIteration)) {
                 // Good run
-                if (!baseMeasurements || theDuration.deviation < baseMeasurements.deviation * 3) {
+                if (!baseMeasurements || theDuration.deviation < baseMeasurements.deviation * 2.5) {
```

Comment:
This change will force additional attempts if the deviation between the base run and current comparison run are too far apart. Previous attempts (prior to check-in) at * 2, caused timeouts from excessive attempts getting executed, so changing to 2.5 as *3 is also causing failures because comparison runs are being hit by garbage / cpu exhaustion (VM build server) -- tests are still running fine locally.